torch_xla/csrc/runtime/stablehlo_helper.cc

```diff
@@ -37,13 +37,23 @@ static std::string getMlirModuleStr(mlir::ModuleOp& mlir_module) {
   return txt_mlir_module;
 }
 
-static std::string getMlirModuleBytecode(const mlir::ModuleOp& mlir_module) {
+static std::string getMlirModuleBytecode(mlir::ModuleOp& mlir_module) {
+  static bool from_pretty_print = runtime::sys_util::GetEnvBool(
```

Comment:
This is to workaround a StableHLO bytecode serialization issue at https://github.com/openxla/stablehlo/issues/1812. If the bytecode is serialized from in-memory `mlir:ModuleOp`, and there are `stablehlo.uniform_quantize/dequantize` in the StableHLO graph, the bytecode serialization would fail, however, txt format serialization can success. The workaround here is to serialized `mlir::ModuleOp` to txt format first, then deserialize to `mlir::ModuleOp`, then serialized to bytecode format.